src/components/character-count/character-count.test.js

```diff
@@ -0,0 +1,400 @@
+          browser = global.__BROWSER__;
+          page = await browser.newPage();
+        });
+
+        afterAll(async () => {
+          await page.close();
+        });
+
+        it('shows the number of characters over the limit', async () => {
+          await goToExample('with-default-value-exceeding-limit');
+          const message = await page.$eval('.hmrc-character-count__message', (el) => el.innerHTML.trim());
+          expect(message).toEqual('You have 23 characters too many');
+        });
+
+        it('shows the number of characters over the limit in Welsh', async () => {
+          await goToExample('with-default-value-exceeding-limit-welsh');
+          const message = await page.$eval('.hmrc-character-count__message', (el) => el.innerHTML.trim());
+          expect(message).toEqual('Mae gennych 23 o gymeriadau yn ormod');
+        });
+
+        it('adds error styles to the textarea', async () => {
+          await goToExample('with-default-value-exceeding-limit');
+          const textareaClasses = await page.$eval('.hmrc-js-character-count', (el) => el.className);
+          expect(textareaClasses).toContain('govuk-textarea--error');
+        });
+
+        it('adds error styles to the count message', async () => {
+          await goToExample('with-default-value-exceeding-limit');
+          const messageClasses = await page.$eval('.hmrc-character-count__message', (el) => el.className);
+          expect(messageClasses).toContain('govuk-error-message');
+        });
+      });
+
+      describe('when a threshold is set', () => {
+        beforeAll(async () => {
+          // eslint-disable-next-line no-underscore-dangle
+          browser = global.__BROWSER__;
+          page = await browser.newPage();
+          await goToExample('with-threshold');
+        });
+
+        afterAll(async () => {
+          await page.close();
+        });
+
+        it('does not show the limit until the threshold is reached', async () => {
+          const visibility = await page.$eval('.hmrc-character-count__message', (el) => window.getComputedStyle(el).visibility);
+          expect(visibility).toEqual('hidden');
+
+          // Ensure threshold is hidden for users of assistive technologies
+          const ariaHidden = await page.$eval('.hmrc-character-count__message', (el) => el.getAttribute('aria-hidden'));
+          expect(ariaHidden).toEqual('true');
+        });
+
+        it('becomes visible once the threshold is reached', async () => {
+          await page.type('.hmrc-js-character-count', 'A'.repeat(8));
+
+          const visibility = await page.$eval('.hmrc-character-count__message', (el) => window.getComputedStyle(el).visibility);
+          expect(visibility).toEqual('visible');
+
+          // Ensure threshold is visible for users of assistive technologies
+          const ariaHidden = await page.$eval('.hmrc-character-count__message', (el) => el.getAttribute('aria-hidden'));
+          expect(ariaHidden).toBeFalsy();
```

Comment:
For consistency can you use toEqual here?

Reply:
@timsb This one is a direct port from `govuk-frontend`: https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/components/character-count/character-count.test.js#L141

Reply:
No problem. There might be a reason why they're specifically using Falsy